src/components/Fetch/Bookmarks.js

```diff
@@ -1,17 +1,20 @@
+// @flow strict
+
 import React from 'react';
-import PropTypes from 'prop-types';
 
-export class Bookmarks extends React.PureComponent {
-  static propTypes = {
-    bookmarks: PropTypes.arrayOf(PropTypes.shape({
-      lineNumber: PropTypes.number.isRequired
-    })).isRequired,
-    setScroll: PropTypes.func.isRequired
-  }
+type lineNumber = {|
```

Comment:
in src/actions/logviewer.js we have `type Bookmark`. I'd recommend moving that to src/models.js and using it here.